scripts/policycheck.py

```diff
@@ -0,0 +1,36 @@
+#!/bin/python
+
+import hashlib
+import sys
+from profanity_check import predict_prob
+
+def main(argv):
+  min_prob = float(sys.argv[1])
+  ret = 0
+
+  with open(sys.argv[2], 'r') as f:
+    ignoreds = f.readlines()
+  ignoreds = [v.strip() for v in ignoreds]
+  print(ignoreds)
+
+  for fn in sys.argv[3:]:
+    with open(fn, 'r') as f:
+      lines = f.readlines()
+      ps = predict_prob(lines)
+      for i in range(len(lines)):
+        if ps[i] >= min_prob:
+          keysrc = "{}\n{}".format(fn, lines[i]).encode()
+          key = hashlib.sha256(keysrc).hexdigest()
+          if key not in ignoreds:
+            ret = ret + 1
+            print("\n\n\n❌ L{} of {}:\n{}".format(i + 1, fn, lines[i]), file=sys.stderr)
+            print("ℹ️ Add {} to {} to ignore this issue".format(key, sys.argv[2]), file=sys.stderr)
+          else:
```

Comment:
Do we want to consider pruning the list of ignored lines to remove any entries that no longer exist in the repo? Depending on how strict the profanity checker is, I could see that list growing very long over time, making it more difficult to tell what should actually be ignored. That is another nit though, and definitely not a blocker here

Reply:
I think best to file an issue and revisit when we have more data.  Same with adjusting the threshold later.

Reply:
#1010 1011